argilla-server/src/argilla_server/api/handlers/v1/users.py

```diff
@@ -26,13 +25,18 @@
 from argilla_server.database import get_async_db
 from argilla_server.models import User
 from argilla_server.security import auth
+from argilla_server.telemetry import TelemetryClient, get_telemetry_client
 
 router = APIRouter(tags=["users"])
 
 
 @router.get("/me", response_model=UserSchema)
-async def get_current_user(request: Request, current_user: User = Security(auth.get_current_user)):
-    await telemetry.track_login(request, current_user)
+async def get_current_user(
+    request: Request,
+    current_user: User = Security(auth.get_current_user),
+    telemetry_client: TelemetryClient = Depends(get_telemetry_client),
+):
+    await telemetry_client.track_user_login(request, current_user)
```

Comment:
@frascuchon was there a reason to include the header along with user/login operations? otherwise I will simplify this a bit and include the user/login as user/read and include `is_login` as metadata.